WcaOnRails/app/controllers/delegates_controller.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+class DelegatesController < ApplicationController
+  before_action :authenticate_user!
```

Comment:
This is probably a separate PR, and maybe controversial, but I would break out an `AuthenticatedUserControllerBase` base class that does `before_action :authenticate_user!`. There are probably a number of utility functions that should end up there too.